### PR TITLE
[cs231n] Lecture 9 complete

### PR DESCRIPTION
**Pull request by bellatoris**
haha

### CODE REVIEW

File: Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md [hunk 1]
```diff
@@ -1,12 +1,97 @@
-## Understanding ConvNet
-### Visualize the filters/kernels
-The parameters of each CONV layer composed of multiple 2D filters. So we can visualize each 2D filter as a image. But only parameters in first CONV layer are meaningful. That's because only input to first CONV layer(which is input image) has graphical meaning in 2D space.
-### Visualize representation
-In every CNN, their last layer is a classifier. And we call input vector to this layer **"code"**. For example, code of VGGNet has 4096 dimension.
-#### t-SNE visualization
+# Understanding ConvNet
```
Comment (chang12) [inline Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md:1]: Could you please remove single-# header? In this document, it's a only single-# header so it does not contribute to entire strucutre of this document. And I also think that the content of it just duplicate the title of directory, "Understanding_and_Visualizing_Convolutional_Neural_Networks".

Comment (bellatoris) [reply Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md:1]: No i dont want

File: Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md [hunk 2]
```diff
@@ -1,12 +1,97 @@
...
+
+> primary cause of neural networks' vulnerability to adversarial perturbation is their **linear nature**.
+
+**Linear nature** of the functions that were used in the forward pass. Images are super high dimensional objects, lots of piexel so there is a hundred and fifty thousand dimensional space. And the real images that we actually trained on are statistical structures and they are constrained to a tiny manifold of this space. ConvNet were very well on that tiny manifold where statistics of images are image-like. But the outsides of the manifold were kind of casting shadows of complete randomness and chaos because we haven't trained there. 
+That is, after ConvNet is completed by searching for a very restrictive manifold in very high dimensional space, you will experience tremendous chaos in shadow that has not yet been searched (e.g. random noise).
```
Comment (chang12) [inline Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md:93]: typo: piexel -> pixels

Comment (bellatoris) [reply Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md:93]: Actually you can fix it

File: Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md (hunk 1 above)
Comment (chang12) [reply Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md:1]: Okay, then.

File: Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md [hunk 3]
```diff
@@ -1,12 +1,97 @@
...
+
+Also, since the input dimesions are so large that even very small element-wise variations are present, the sum is so large that the results are very severe. (fool binary linear classifier를 설명하긴 싫은데, 이 문장은 넣고 싶네요 ㅜㅜ 근데 뭔가 뜬금 없어 보여서 어떻게 해야 넣어야 할지 모르겠습니다.)
+
```
Comment (chang12) [inline Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md]: Um.... I don't feel it sudden, but if you feel uncomfortable, then what about appending informative picture to implement the sentence?

File: Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md (hunk 2 above)
Comment (chang12) [reply Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md:93]: Oh, sorry.

File: Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md (hunk 3 above)
Comment (chang12) [inline Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md:96]: Um.... I don't feel it sudden, but if you feel uncomfortable, then what about appending informative picture to implement the sentence?

**Comment by chang12** [comment]
I finished my review @bellatoris ~

File: Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md (hunk 3 above)
Comment (bellatoris) [reply Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md:96]: This comment is deleted, cause you modified the markdown. And if you want to append the picture, plz do that

**Comment by bellatoris** [comment]
I finished feedback of comments ~ @chang12

**Comment by bellatoris** [comment]
So is there any comments about the contents?

**Comment by chang12** [comment]
@bellatoris 
Nope~ It looks great to me.

**Pull request merged**